### PR TITLE
Dev Tooling: Better mobile support

### PR DESCRIPTION
**Pull request by jeremywiebe**
## Summary:

This PR improves our mobile support for our dev tooling (both Storybook and the custom Dev UI):

  * Adds the `perseus-mobile` class to the parent div of the renderers. Some of our Perseus styling depends on an ancestor node having the `perseus-mobile` class in order to render for mobile.
  * Adds a mobile switch to the `RendererWithDebugUI` component so we can manually flip to mobile rendering in Storybook (already exists in the Dev UI)

Finally, I've changed the`SideBySide` component slightly so that the right panel will move below the left panel on narrower screens. This will make inspecting a story with mobile emulation on much cleaner (the right panel, usually the Perseus JSON, would overlap the left panel previously).

### Dev UI (with mobile)

image

### Storybook (with mobile) 

image

### Inspecting a Story with mobile emulation enabled

image

Issue: "none"

## Test plan:

### CODE REVIEW

File: dev/gallery.tsx [hunk 1]
```diff
@@ -107,7 +107,10 @@ export function Gallery() {
                 </nav>
             </header>
             <main className={css(styles.main)}>
-                <View style={styles.cards}>
+                <View
+                    style={styles.cards}
+                    className={isMobile ? "perseus-mobile" : ""}
```
Comment (jeremywiebe) [inline dev/gallery.tsx:112]: This is the magic to flip Perseus into mobile rendering for some widgets (used in our `.less` files mostly).

File: testing/renderer-with-debug-ui.tsx [hunk 2]
```diff
@@ -28,22 +31,42 @@ export const RendererWithDebugUI = ({
...
+            leftTitle={
+                <View
+                    style={{
+                        flexDirection: "row",
+                        alignItems: "center",
+                        width: "100%",
+                    }}
+                >
+                    Widget
+                    <View style={{marginLeft: "auto"}}>
```
Comment (jeremywiebe) [inline testing/renderer-with-debug-ui.tsx:47]: This was the only way I could figure out pushing the switch to the right side.  `alignSelf: 'end'` didn't work. 😞

Comment (benchristel) [reply testing/renderer-with-debug-ui.tsx:47]: `marginLeft` is a fine solution. Another thing I often do in flexbox layouts is add a blank element between the left-aligned and right-aligned things, and set it to `flexGrow: 1` so it expands to fill the space.

File: testing/side-by-side.tsx [hunk 3]
```diff
@@ -39,15 +40,18 @@ const SideBySide = ({
...
     sideBySide: {
         display: "flex",
+        flexWrap: "wrap",
         flexDirection: "row",
     },
     leftPanel: {
         paddingRight: "30px",
-        flexGrow: 1,
+        flexBasis: "400px",
+        margin: Spacing.medium_16,
```
Comment (benchristel) [inline testing/side-by-side.tsx:49]: Do we need both padding and margin on this element?

File: testing/side-by-side.tsx [hunk 4]
```diff
@@ -39,15 +40,18 @@ const SideBySide = ({
...
     },
     rightPanel: {
         flexGrow: 1,
         maxWidth: "50%",
+        flexBasis: "400px",
```
Comment (benchristel) [inline testing/side-by-side.tsx:54]: I wonder if this `400px` magic number should be extracted to a constant. I suspect there are other places where we reference 400px, too.

**Review by benchristel — APPROVED** [review]
LGTM! Thanks!

I left a couple comments inline.

File: testing/side-by-side.tsx (hunk 3 above)
Comment (jeremywiebe) [reply testing/side-by-side.tsx]: It wasn't working as I intended so I've tweaked the styles to be simpler.

File: testing/side-by-side.tsx (hunk 4 above)
Comment (jeremywiebe) [reply testing/side-by-side.tsx]: I found a few places that had a 400 constant already.  I've refactored to use one of them.  I think 400 is a graph constant mostly, but for our stories I think it mostly makes sense for now.

**Pull request merged**